ui/design/src/components/TopBar/index.tsx

```diff
@@ -128,6 +136,11 @@ const Topbar: React.FC<ITopbarProps> = props => {
     }
   }, [otherAreaItems]);
 
+  const shouldRenderOnboarding = React.useMemo(
```

Comment:
You should use react-router!

Reply:
@SeverS I can use the <Route> component for conditional rendering in the top bar, but I sort of still need this flag to pass down to the ProfileMenu to avoid rendering moderation menu options 🤔

Reply:
I've reviewed the designs once again and i think the moderation history link should remain in place, so no need for that flag in the Profile Menu.